contracts/vaults/balancer/internal/strategy/StrategyUtils.sol

```diff
@@ -73,6 +60,19 @@ library StrategyUtils {
             params.exchangeData
         );
 
+        // stETH generally has deeper liquidity than wstETH, setting tradeUnwrapped
+        // to lets the contract trade in stETH instead of wstETH
+        if (params.tradeUnwrapped && sellToken == address(Deployments.WRAPPED_STETH)) {
```

Comment:
This if statement could be nested such that:
```
if (params.tradeUnwrapped) {
  if (sellToken == wstETH) {

  } else if (buyToken == wstETH) {

  }
}

Reply:
fixed